content/docs/1_guide/16_emails/guide.txt

```diff
@@ -265,6 +265,27 @@ $kirby->email([
 ]);
 ```
 
+It can be useful to add embedded images:
+
+```php "/site/templates/emails/email.html.php"
+<img src="cid:image">
+<!-- Email content -->
+```
+
+```php
+$kirby->email([
+    // …
+		'template' => 'email',
+    'beforeSend' => function ($mailer) {
+				// $image should return file path
+				$image = asset('assets/images/logo.png')->root();
```

Comment:
This looks wrong, $image in this line returns a string, then you cannot call `$image->filename()`. This should probably read:

```
$image = asset('assets/images/logo.png');
$mailer->AddEmbeddedImage($image->root(), 'image', $image->filename(), 'base64', $image->mime());
```

Reply:
Yep, my bad. Next commit has fixed that 👍